src/pages/Detail/DetailTop.js

```diff
@@ -61,9 +65,16 @@ export class DetailTop extends Component {
         console.log(response.status);
       })
       .then(response => {
-        alert('장바구니에 추가되었습니다.');
+        this.onModal();
       });
   };
+  //
+  goToCart = () => {
+    this.addCart();
+    setTimeout(() => {
+      window.location.href = 'http://localhost:3000/cart';
```

Comment:
- 여기도 수정이 되어야 할 것 같네요!

Reply:
```suggestion
      window.location.href = 'http://localhost:3000/cart';
```
```suggestion
      setTimeout(() => {
      this.props.history.push('http://localhost:3000//cart');
    }, 1000);
```

해당 부분 이렇게 바꿧는데 오류가 뜹니다 혹시 cart 페이지로 이동시키고 싶은데 어떻게 이동시켜야 하나요?

Reply:
아...withRouter를 안써서 그런거엿군요 수정했습니다!